theano/gpuarray/type.py

```diff
@@ -24,22 +24,36 @@
 _context_reg = {}
 
 
+def do_gpu_support(data):
```

Comment:
I would name this gpu_supported, since do_gpu_support is an action, not a question.